### DISCUSSION TITLE
FieldNotFound within json

### DISCUSSION THREAD

**Discussion by Gregoor** [prisma/prisma#12246] [Q&A] [answered]
I think this is a bug, but I do not have a local reproduction yet so I will post it here for now. So the bug we are seeing is:
> Error in query graph construction:
> `DomainError(FieldNotFound { name: "some", container_type: "model", container_name: "user_slack_installation" })`

There is no error stack attached to this but the query I am suspecting is the following:

```ts
db.user.findFirst({
  where: { user_slack_installation: { some: { data: { path: ["user", "id"], equals: slackUserId } } } },
});
```

Now I am assuming there might be something wrong with the JSON here for certain queries (again, I unfortunately do not have a repro yet), but that I get prisma error without a stack, instead of an empty result is probably a bug, no?

**Comment by janpio** [comment, marked as answer]
Yes, and it shows that something unexpected is going wrong in a pretty bad way. So if you get any more information, definitely make this a real bug report issue and provide all the information you have available. We definitely want to look into this.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
